pkg/report/report.go

```diff
@@ -353,8 +363,10 @@ func Generate(ctx context.Context, path string, mrs yara.MatchRules, ignoreTags
 				}
 			case "author_url":
 				b.RuleAuthorURL = v
-			case "__bincapz__":
-				fr.IsBincapz = true
+			case fmt.Sprintf("__%s__", BINARY):
+				if v == "true" {
```

Comment:
Just a quick check to ensure that the metadata value is actually `"true"` instead of solely relying on the presence of the `__bincapz__` identifier.